orderable/models.py

```diff
@@ -134,6 +134,13 @@ def save(self, *args, **kwargs):
         # Call the "real" save() method.
         super(Orderable, self).save(*args, **kwargs)
 
+    def _pass_through_save(self, *args, **kwargs):
```

Comment:
^ Done this properly now (I edited my local site packages version rather than the checked-out code lol)